exec/vector/src/main/java/org/apache/drill/exec/record/metadata/AbstractColumnMetadata.java

```diff
@@ -295,6 +285,8 @@ public String toString() {
         .toString();
   }
 
+  protected void appendContents(StringBuilder buf) { }
+
   @JsonProperty("type")
   @Override
   public String typeString() {
```

Comment:
Better make this abstract since all columns which implement this, must ensure parser can parse string back to type.